src/components/Button/Button.scss

```diff
@@ -133,18 +133,18 @@
   align-items: center;
   background-color: transparent;
   border: 0;
-  display: flex;
   padding: 0;
+  position: relative;
 
   .ms-Button-icon {
     color: $ms-color-themePrimary;
     display: inline-block;
     font-size: $ms-font-size-s;
     margin-right: 4px;
-    position: relative;
-    top: 1px;
+    padding-top: 5px;
     text-align: center;
 
+
```

Comment:
Small nit, but is there a reason for this extra whitespace? If not I'd probably take it out.